package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "react-native-fetch-api",
   "description": "A fetch API polyfill for React Native with text streaming support.",
-  "version": "2.0.0",
+  "version": "3.0.0",
```

Comment:
Please revert this change. The version is bumped automatically during the release process.